bug/analysis settings for `test_custom_model_runner_does_not_exist___generate_losses_is_called_output_files_are_tared_up` invalid

ODS Tools PR https://github.com/OasisLMF/ODS_Tools/pull/271 requires at least 1 analysis output and required `gul_summaries` to at least have 1 item. 

Causes the following test to fail: 

```
FAILED src/model_execution_worker/tests/test_tasks.py::StartAnalysis::test_custom_model_runner_does_not_exist___generate_losses_is_called_output_files_are_tared_up - ods_tools.oed.common.OdsException:
``` 

with the following error: 

```
E           ods_tools.oed.common.OdsException:
E           JSON Validation error in 'analysis_settings.json': {
E               "no output": [
E                   "no output selected, please enable at least one output"
E               ],
E               "analysis_settings_schema gul_summaries": [
E                   "[] should be non-empty"
E               ]
E           }

../ODS_Tools/ods_tools/oed/settings.py:438: OdsException
```

--- a/src/model_execution_worker/tests/test_tasks.py
+++ b/src/model_execution_worker/tests/test_tasks.py
@@ -102,8 +102,8 @@ class StartAnalysis(TestCase):
                     },
                     "model_name_id": "PiWind",
                     "model_supplier_id": "OasisLMF",
-                    "gul_output": False,
-                    "gul_summaries": []
+                    "gul_output": True,
+                    "gul_summaries": [{'id': 1}]
                 }
 
                 with open(settings_file_path, 'w') as f:

